src/test/java/org/lsc/plugins/connectors/james/integration/IdentitySynchronizationIntegrationTest.java

```diff
@@ -0,0 +1,148 @@
+
+        int MAPPED_JAMES_WEBADMIN_PORT = jamesContainer.getMappedPort(JAMES_WEBADMIN_PORT);
+
+        requestSpecification = new RequestSpecBuilder().setPort(MAPPED_JAMES_WEBADMIN_PORT)
+            .setContentType(ContentType.JSON).setAccept(ContentType.JSON)
+            .setConfig(newConfig().encoderConfig(encoderConfig().defaultContentCharset(StandardCharsets.UTF_8)))
+            .addHeader("Authorization", "Bearer " + jwtToken())
+            .setBasePath("")
+            .build();
+        RestAssured.enableLoggingOfRequestAndResponseIfValidationFails();
+
+        given(requestSpecification).basePath("/domains").put(DOMAIN).then().statusCode(HttpStatus.SC_NO_CONTENT);
+    }
+
+    @AfterEach
+    void tearDown() {
+        jamesContainer.close();
+        ldapContainer.close();
+    }
+
+    private GenericContainer<?> createLdap(Network network) {
+        return new GenericContainer<>(
+            new ImageFromDockerfile()
+                .withFileFromClasspath("populate.ldif", "prepopulated-ldap/populate.ldif")
+                .withFileFromClasspath("Dockerfile", "prepopulated-ldap/Dockerfile"))
+            .withNetworkAliases("ldap")
+            .withNetwork(network)
+            .withEnv("SLAPD_DOMAIN", "james.org")
+            .withEnv("SLAPD_PASSWORD", "mysecretpassword")
+            .withEnv("SLAPD_CONFIG_PASSWORD", "mysecretpassword")
+            .withCreateContainerCmdModifier(createContainerCmd -> createContainerCmd.withName("team-mail-openldap-testing" + UUID.randomUUID()));
+    }
+
+    private String jwtToken() throws Exception {
+        Security.addProvider(new BouncyCastleProvider());
+
+        RSAPublicKey publicKey = getPublicKey();
+        RSAPrivateKey privateKey = getPrivateKey();
+
+        Algorithm algorithm = Algorithm.RSA256(publicKey, privateKey);
+        String jwtToken = JWT.create()
+            .withSubject("admin@james.org")
+            .withClaim("admin", true)
+            .withIssuedAt(Date.from(Instant.now()))
+            .sign(algorithm);
+        System.out.println("JWT TOKEN: " + jwtToken);
+        return jwtToken;
+    }
+
+    private RSAPublicKey getPublicKey() throws Exception {
+        try (PEMReader pemReader = new PEMReader(new FileReader(PUBLIC_KEY.getFile()))) {
+            Object readObject = pemReader.readObject();
+            return (org.bouncycastle.jce.provider.JCERSAPublicKey) readObject;
+        }
+    }
+
+    private RSAPrivateKey getPrivateKey() throws Exception {
+        try (PEMReader pemReader = new PEMReader(new FileReader(PRIVATE_KEY.getFile()), "james"::toCharArray)) {
+            Object readObject = pemReader.readObject();
+            return (RSAPrivateKey) ((java.security.KeyPair) readObject).getPrivate();
+        }
+    }
+
+    // TODO write test running the lsc job connect TMail and LDAP
+    @Disabled("This is for manual test now. Could give it more effort to be a automation test")
+    @Test
```

Comment:
Manual test (local):
Given we have Alice and Bob entries in LDAP.
Then LSC identity job should create default identity for Alice and BOB:
- LSC:
![image](https://user-images.githubusercontent.com/55171818/223379120-f6e694dc-041e-4449-b5ff-2336641e9028.png)
- James get identity for BOB:
![image](https://user-images.githubusercontent.com/55171818/223381402-8ec8084a-5d13-4698-a282-e6fca0ac3631.png)